packages/client/bin/cli.ts

```diff
@@ -608,36 +632,38 @@ async function run() {
   ensureDirSync(configDirectory)
   const key = await Config.getClientKey(datadir, common)
   logger = getLogger(args)
+  const bootnodes = args.bootnodes ? parseMultiaddrs(args.bootnodes) : undefined
+  const multiaddrs = args.multiaddrs ? parseMultiaddrs(args.multiaddrs) : undefined
   const config = new Config({
+    accounts,
+    bootnodes,
     common,
-    syncmode: args.syncmode,
-    lightserv: args.lightserv,
     datadir,
-    key,
-    transports: args.transports,
-    bootnodes: args.bootnodes ? parseMultiaddrs(args.bootnodes) : undefined,
-    port: args.port,
-    extIP: args.extIP,
-    multiaddrs: args.multiaddrs ? parseMultiaddrs(args.multiaddrs) : undefined,
-    logger,
-    saveReceipts: args.saveReceipts,
-    txLookupLimit: args.txLookupLimit,
-    maxPerRequest: args.maxPerRequest,
-    minPeers: args.minPeers,
-    maxPeers: args.maxPeers,
-    dnsAddr: args.dnsAddr,
-    dnsNetworks: args.dnsNetworks,
     debugCode: args.debugCode,
     discDns: args.discDns,
     discV4: args.discV4,
+    dnsAddr: args.dnsAddr,
+    dnsNetworks: args.dnsNetworks,
+    extIP: args.extIP,
+    key,
+    lightserv: args.lightserv,
+    logger,
+    maxPeers: args.maxPeers,
+    maxPerRequest: args.maxPerRequest,
     mine: args.mine || args.dev,
-    accounts,
     minerCoinbase: args.minerCoinbase,
+    minPeers: args.minPeers,
+    multiaddrs,
+    port: args.port,
+    saveReceipts: args.saveReceipts,
+    syncmode: args.syncmode,
+    transports: args.transports,
+    txLookupLimit: args.txLookupLimit,
   })
   config.events.setMaxListeners(50)
 
-  const client = await runNode(config)
-  const servers = args.rpc || args.rpcEngine ? runRpcServers(client, config, args) : []
+  const client = await startClient(config)
+  const servers = args.rpc || args.rpcEngine ? startRPCServers(client) : []
 
   process.on('SIGINT', async () => {
     config.logger.info('Caught interrupt signal. Shutting down...')
```

Comment:
Whew.

That's a really great and much needed clean-up on the CLI script!!! 😄 👍 

Nice.

🎉